integration_tests/mockApis/auth.ts

```diff
@@ -161,28 +141,13 @@ const stubProfile = (roles = [], userId = defaultUserId, isActive = true, apArea
       headers: {
         'Content-Type': 'application/json;charset=UTF-8',
       },
-      jsonBody: {
-        ...defaultProfile,
+      jsonBody: userFactory.build({
+        name,
```

Comment:
and then here we could do `userFactory.build({...user})`

Reply:
Good plan!